test/modules/aws-account-integration.ts

```diff
@@ -14,8 +14,8 @@ const query = runQuery.bind(null, dbAlias);
 const sync = runSync.bind(null, dbAlias);
 
 jest.setTimeout(360000);
-beforeAll(execComposeUp);
-afterAll(() => execComposeDown());
+beforeAll(async () => await execComposeUp());
+afterAll(async () => await execComposeDown());
```

Comment:
This is a useless change. `async` makes the function return a promise, which is what `execComposeUp` already returns. You can revert all of these and it'll still work perfectly fine, just like it did before.

Reply:
Well, no. `afterAll` was makes Jest hang up with an error that not all async operations completed when you run it for a single test. Feel free to try it

Reply:
https://jestjs.io/docs/api#afterallfn-timeout

The docs do not mention an argument parameter, so it should work just fine. I'm guessing some other change happened at the same time and was misattributed to this change for fix?

Reply:
https://github.com/iasql/iasql-engine/pull/542#discussion_r831152060